modules/system/tests/bootstrap/PluginManagerTestCase.php

```diff
@@ -52,7 +51,7 @@ public function setUp() : void
         UpdateManager::forgetInstance();
 
         // Forces plugin migrations to be run again on every test
-        VersionManager::forgetInstance();
+//        PluginVersionManager::forgetInstance();
```

Comment:
Is this temporary?

Reply:
Yes, it's a note to self that I need to look into how not flushing the instance works inside tests